.rosinstall

```diff
@@ -0,0 +1,30 @@
+- git:
```

Comment:
What is this `.rosinstall` for?
Could you write usage of this on README?

Reply:
This file is same as `.travis.rosinstall` of `master`.
I think we don't need this file, and I don't know the usage of some packages like `https://svn.code.sf.net/p/jsk-ros-pkg/code/trunk/openrave_planning/collada_robots`.
But I could not erase this file because these URL may be needed in the future.

Reply:
I see. How about just commenting out unnecessary lines of `.travis.rosinstall` and adding comments like `What is for?`
I think jsk_model_tools, jsk_roseus, and collada_robots are added at #36 .
And nowadays, we can use jsk_model_tools and jsk_roseus from apt, so they are no more required.
@k-okada Do you remember why you added `collada_robots`?
I searched source code at c5ea4a4960be78, but I couldn't find about `collada_robots`.

Reply:
Thank you. I fixed it.

Reply:
@k-okada 
> @k-okada Do you remember why you added `collada_robots`?
> I searched source code at [c5ea4a4](https://github.com/start-jsk/rtmros_gazebo/commit/c5ea4a4960be78075156a416e8cfed70ad4a70fb), but I couldn't find about `collada_robots`.

Sorry, but do you have any comment about this?